libs/common-services/src/test/java/uk/gov/di/ipv/core/library/fixtures/VcFixtures.java

```diff
@@ -0,0 +1,890 @@
+package uk.gov.di.ipv.core.library.fixtures;
+
+import uk.gov.di.ipv.core.library.domain.BirthDate;
+import uk.gov.di.ipv.core.library.domain.NameParts;
+import uk.gov.di.ipv.core.library.enums.Vot;
+import uk.gov.di.ipv.core.library.helpers.TestVc;
+
+import java.time.Instant;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import static uk.gov.di.ipv.core.library.domain.VerifiableCredentialConstants.CRI_STUB_CHECK_EVIDENCE_TYPE;
+import static uk.gov.di.ipv.core.library.domain.VerifiableCredentialConstants.RISK_ASSESSMENT_CREDENTIAL_TYPE;
+import static uk.gov.di.ipv.core.library.domain.VerifiableCredentialConstants.RISK_ASSESSMENT_EVIDENCE_TYPE;
+import static uk.gov.di.ipv.core.library.domain.VerifiableCredentialConstants.VC_FAMILY_NAME;
+import static uk.gov.di.ipv.core.library.domain.VerifiableCredentialConstants.VC_GIVEN_NAME;
+import static uk.gov.di.ipv.core.library.domain.VerifiableCredentialConstants.VC_NAME_PARTS;
+import static uk.gov.di.ipv.core.library.domain.VerifiableCredentialConstants.VERIFIABLE_CREDENTIAL_TYPE;
+import static uk.gov.di.ipv.core.library.helpers.VerifiableCredentialGenerator.generateVerifiableCredential;
+
+public interface VcFixtures {
+    List<TestVc.TestEvidence> SUCCESSFUL_EVIDENCE =
+            List.of(TestVc.TestEvidence.builder().strengthScore(4).validityScore(2).build());
+    List<TestVc.TestEvidence> UNSUCCESSFUL_EVIDENCE =
+            List.of(TestVc.TestEvidence.builder().strengthScore(4).validityScore(0).build());
+    List<TestVc.TestEvidence> D02_CI_EVIDENCE =
+            List.of(
+                    TestVc.TestEvidence.builder()
+                            .strengthScore(4)
+                            .validityScore(2)
```

Comment:
With this CI, I think the validity here would be `0`

Reply:
I don't know if that's _necessarily_ true if it's a non-breaching CI, but it's less common certainly.